nemo_aligner/models/nlp/gpt/megatron_gpt_dpo_model.py

```diff
@@ -238,9 +306,44 @@ def split_output_tensor(self, output_tensor):
         chosen_logps, reject_logps = torch.split(output_tensor.float(), len(output_tensor) // 2, dim=0)
         return chosen_logps, reject_logps
 
-    def get_reduced_masked_logps(self, logps, labels, average_log_probs=False):
+    def get_reduced_masked_logps(self, logps, labels, cu_seqlens=None, average_log_probs=False):
         assert logps.shape == labels.shape, "logps and labels shape mismatch"
 
+        ## mbs = 1
```

Comment:
is this assumed? If so, can this just be made into an assertion?

Reply:
I added an assertion later in this file